csrankings-m.csv

```diff
@@ -1940,6 +1940,7 @@ Ming Li 0006,University of Texas at Arlington,https://mentis.uta.edu/explore/pro
 Ming Liu 0001,HKUST,http://www.ece.ust.hk/ece.php/profile/facultydetail/eelium,qexey-UAAAAJ
 Ming Liu 0004,Harbin Institute of Technology,http://homepage.hit.edu.cn/liuming1981,VJtmTREAAAAJ
 Ming Liu 0027,University of Wisconsin - Madison,https://mgliu.sites.cs.wisc.edu,_JNse44AAAAJ
+Min Lu 0002,Shenzhen University,http://deardeer.github.io/

```

Comment:
Missing Google Scholar ID.

Reply:
Oh, sorry I missed that... 
My Google Scholar ID is: xFAU798AAAAJ
Should I make a new submission?

Reply:
You can just update the entry and commit + push to your branch.

Reply:
Thanks : ) 
I have updated it.